app/controllers/atmosphere/admin/virtual_machine_templates_controller.rb

```diff
@@ -17,13 +21,26 @@ def show
   def edit
   end
 
+  # GET /virtual_machine_templates/1/share
+  def share
+  end
+
+  # POST /virtual_machine_templates/1/export
+  def export
```

Comment:
I think the history could be truncated after @mkasztelnik changed Atmo into an engine. But perhaps this is about ath like that: https://github.com/dice-cyfronet/atmosphere/blob/master/app/controllers/atmosphere/admin/appliance_configuration_templates_controller.rb